sleepecg/config.py

```diff
@@ -20,14 +19,17 @@
 
 
 def _read_yaml(path: Path) -> Dict[str, Any]:
-    with open(path) as file:
-        cfg = yaml.safe_load(file)
-        # empty .yml-files are loaded as `None`
-        if cfg is None:
-            return {}
-        if not isinstance(cfg, dict):
-            raise ValueError(f'Invalid YAML config file at {_USER_CONFIG_PATH}')
-        return cfg
+    try:
+        with open(path) as file:
+            cfg = yaml.safe_load(file)
+            # empty .yml-files are loaded as `None`
+            if cfg is None:
+                return {}
+            if not isinstance(cfg, dict):
+                raise ValueError(f'Invalid YAML config file at {_USER_CONFIG_PATH}')
+            return cfg
+    except FileNotFoundError:
+        return {}
```

Comment:
Hm, in this case there's a lot going on in the `try` block, which maybe means that it would be better to just check if the file exists first?

Reply:
oh right, most of that doesn't even need to be inside the `with`, fixed in 327a978:)